### PR TITLE
override Chakra styling changes in DoenetML

### LINKED ISSUE #1900: Adjust height of <aside> and boxed sections to a reasonable width
Current styling is awkwardly oversized and interrupts the flow of content.
image

### PR DESCRIPTION
**Pull request by dqnykamp**
Fixes #1900

### CODE REVIEW

File: src/Tools/cypressTest/index.html [hunk 1]
```diff
@@ -26,56 +26,55 @@
...
- 
-  @font-face {
-    font-family: 'Open Sans';
-    font-style: normal;
-    font-weight: 700;
-    src:
-      url('/fonts/files/open-sans-v18-latin-700.woff2') format('woff2'),
-      url('/fonts/files/open-sans-v18-latin-700.woff') format('woff');
-  }
+      url(OpenSansRegularWoff) format('woff'); */ url('/fonts/files/open-sans-v18-latin-regular.woff2')
```
Comment (dqnykamp) [inline src/Tools/cypressTest/index.html:29]: Note: the only non-whitespace changes start on line 100

File: src/Tools/test/index.html [hunk 2]
```diff
@@ -1,78 +1,84 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
```
Comment (dqnykamp) [inline src/Tools/test/index.html:3]: Note: the only non-whitespace changes start on line 127.

File: index.html [hunk 3]
```diff
@@ -120,12 +123,72 @@
...
+        margin-block-end: 1.67em;
+      }
+      .doenet-viewer p {
+        margin-block-start: 1em;
+        margin-block-end: 1em;
+      }
+      .doenet-viewer input {
+        box-sizing: content-box;
+        padding: 2px 1px;
+      }
```
Comment (Lyanthropos) [inline index.html:168]: For future reference, you can next all of these within the first .doenet-viewer selector for a little less repetition.

**Review by Lyanthropos — APPROVED** [review]
One closer to deployment

**Review by jaltekruse — APPROVED** [review]
+1

**Pull request merged**